docker-compose.yml

```
@@ -335,6 +335,8 @@ services:
     environment:
       HEDERA_MIRROR_WEB3_DB_HOST: db
       HEDERA_MIRROR_WEB3_EVM_NETWORK: OTHER
+      HEDERA_MIRROR_WEB3_EVM_MODULARIZEDSERVICES: true
+      HEDERA_MIRROR_WEB3_EVM_MODULARIZEDTRAFFICPERCENT: "1"
```

I wonder 🤔  if we really want to HardCode those values, or we want to be able to parametrize them using `.env` file and end up with something like:

```suggestion
      HEDERA_MIRROR_WEB3_EVM_MODULARIZEDSERVICES: ${MIRROR_WEB3_EVM_MODULARIZEDSERVICES}
      HEDERA_MIRROR_WEB3_EVM_MODULARIZEDTRAFFICPERCENT: ${MIRROR_WEB3_EVM_MODULARIZEDTRAFFICPERCENT}
```

see this for other .env examples https://github.com/hiero-ledger/hiero-local-node/blob/main/.env#L33